umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md

```diff
@@ -73,6 +68,46 @@ sudo $(which kubectl) scale statefulset {{< resource type="statefulset" name="ka
 sudo $(which kubectl) scale statefulset {{< resource type="statefulset" name="mqttbroker" >}} --replicas=0 -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
 ```
 
+### Copy kubeconfig file
+
+The backup script is located inside the folder you downloaded earlier.
+
+1. Open a terminal and navigate inside the folder.
+
+      ```powershell
+      cd <FOLDER_PATH>
+      ```
+
+2. Create a file with the name `k3s.yaml` on your local device.
+
+      ```powershell
+      cd > k3s.yaml
+      ```
+
+3. Show the content of kubeconfig file with the following command in 
+the instance's shell and copy&paste the output to `k3s.yaml` on your device:
+
+      ```bash
+      sudo cat /etc/rancher/k3s/k3s.yaml
+      ```
+
+      {{% notice tip %}}
+      This guide assumes that the kubeconfig file is located at 
+      `/etc/rancher/k3s/k3s.yaml`. The location depends on your 
+      environment and may differ.
+      {{% /notice %}}
+
+4. Adjust the server's IP address under `clusters` &rArr; `cluster` &rArr; `server`, 
+then save it:
+      ```yaml
+      apiVersion: v1
+      clusters:
+      - cluster:
+         certificate-authority-data: xxx
+         server: https://<your-server-ip>:6443
+      
+      ```
+
```

Comment:
``````markdown
### Copy kubeconfig file

To run the backup script, you'll first need to obtain a copy of the Kubernetes configuration file from your instance. This is essential for providing the script with access to the instance.

1. In the shell of your instance, execute the following command to display the Kubernetes configuration:

   ```bash
   sudo cat /etc/rancher/k3s/k3s.yaml
   ```

   Make sure to copy the entire output to your clipboard.

   {{% notice tip %}}
   This tutorial is based on the assumption that your kubeconfig file is located at /etc/rancher/k3s/k3s.yaml. Depending on your setup, the actual file location might be different.
   {{% /notice %}}

2. Open a text editor, like Notepad, on your local machine and paste the copied content.
3. In the pasted content, find the server field. It usually defaults to `https://127.0.0.1:6443`. Replace this with your instance's IP address

    ```yaml
    server: https://<INSTANCE_IP>:6443
    ```

4. Save the file as `k3s.yaml` inside the `backup` folder you downloaded earlier.
``````

Reply:
updated